### PR TITLE
Settings#2 account info cells

### PR DESCRIPTION
**Pull request by ebarrera95**

### CODE REVIEW

File: FitGoal/FitGoal/Setting/AccountInfoCell.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+//
+
+import UIKit
+
+class AccountInfoCell: UITableViewCell {
+    
+    var cellTitle =  "" {
+        didSet {
+            cellTitleLabel.attributedText = cellTitle.formattedText(font: "Roboto-Light", size: 15, color: #colorLiteral(red: 0.5215686275, green: 0.5333333333, blue: 0.568627451, alpha: 1), kern: 0.3)
+            if cellTitle == "Notifications" {
```
Comment (rlaguilar) [inline FitGoal/FitGoal/Setting/AccountInfoCell.swift]: You shouldn't use the value of the title for this, try to use a specific property for that configuration. Once we add localisation for the app this code won't work anymore.

File: FitGoal/FitGoal/Setting/AccountInfoCell.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    static let identifier = "NotificationsCell"
+    
+    let switchControl: UISwitch = {
+        let switchControl = UISwitch()
+        switchControl.isOn = true
+        switchControl.onTintColor = #colorLiteral(red: 0.5647058824, green: 0.07450980392, blue: 0.9568627451, alpha: 1)
+        return switchControl
+    }()
+    
+    let cellTitleLabel = UILabel()
```
Comment (rlaguilar) [inline FitGoal/FitGoal/Setting/AccountInfoCell.swift]: Private?

File: FitGoal/FitGoal/Setting/SettingsViewController.swift [hunk 3]
```diff
@@ -77,6 +116,11 @@ class SettingsViewController: UITableViewController {
     }
 }
 
+enum SettingsTableViewSection: Int, CaseIterable {
```
Comment (rlaguilar) [inline FitGoal/FitGoal/Setting/SettingsViewController.swift]: Private?

**Review by rlaguilar — CHANGES_REQUESTED** [review]

File: FitGoal/FitGoal/Setting/SettingsViewController.swift [hunk 4]
```diff
@@ -77,9 +116,27 @@ class SettingsViewController: UITableViewController {
...
+    
+    var name: String {
+        switch self {
+        case .notifications:
+            return "Notifications"
+        case .accountInfo:
+            return "Account Info"
+        }
+    }
+}
```
Comment (ebarrera95) [inline FitGoal/FitGoal/Setting/SettingsViewController.swift:142]: So these are the changes that I propose. Downside: It's a new `enum`.  What do you recond? It's almost always the solution that comes to my mind when I face a problem like this.

**Review by rlaguilar — APPROVED** [review]

**Pull request merged**